notes/lecture6.tex

```diff
@@ -0,0 +1,253 @@
+\section{Lecture 6: Discovering acceleration}
+
+In this lecture, we seek to find methods that converge faster than those discussed in previous lectures. To derive this accelerated method, we start by considering the special case of optimizing quadratic functions.
+
+\subsection{Quadratics}
+
+\begin{definition}[Quadratic function] A quadratic function $f\colon\R^n \to \R$ takes the form: 
+\begin{equation*}
+f(x) = \frac{1}{2}x^T A x - b^T x + c,
+\end{equation*}
+where $A \in \mathbb{S}^n$, $b \in \R^n$ and $c \in \R$.
```

Comment:
We haven't use this notation for symmetric matrices, it usually denotes the sphere.

Reply:
I believe lecture 1 introduced some notation for this. If not, just say it's symmetric without introducing notation for the set. Thanks.